src/xhr.js

```diff
@@ -74,7 +74,8 @@ const xhrFactory = function() {
       }
     }
 
-    const xhrMethod = videojs.Vhs.xhr.name !== 'XhrFunction' ? videojs.Vhs.xhr : videojsXHR;
+    // Use the standard videojs.xhr() method unless `videojs.Vhs.xhr` has been overriden
```

Comment:
can we add a comment here that we can go back to use `Function.name` when we drop IE11?
```suggestion
    // Use the standard videojs.xhr() method unless `videojs.Vhs.xhr` has been overriden
    // TODO: switch back to videojs.Vhs.xhr.name === 'XhrFunction' when we drop IE11
```